Add name resolution error solution in FAQ (#116)
metalbear-co/mirrord.dev: content/docs/overview/faq.md

   Please let us know if you're having trouble with SIP by opening an issue on [GitHub](https://github.com/metalbear-co/mirrord) or talking to us on [Discord](https://discord.gg/metalbear).

## My local process fails to resolve the domain name of a Kubernetes service in the same cluster, what could be the issue?

> You probably need to change `feature.fs.mode` from `local` to `localwithoverrides`.

Reviewer: ```suggestion
If you've set `feature.fs.mode` to `local`, try changing it to `localwithoverrides`.
```

> In the `local` mode is on, all file will be opened locally. This might prevent your process from resolving cluster-internal domain names correctly, because it can no longer read Kubelet-generated configuration files like `/etc/resolv.conf`. With `localwithoverrides`, such files are read from the remote pod instead.

Reviewer: ```suggestion
When the `local` mode is set, all files will be opened locally. This might prevent your process from resolving cluster-internal domain names correctly, because it can no longer read Kubelet-generated configuration files like `/etc/resolv.conf`. With `localwithoverrides`, such files are read from the remote pod instead.
```
